src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -4487,7 +4478,18 @@ internal static object EventListenersLock
             get
             {
                 if (s_EventSources == null)
+                {
                     Interlocked.CompareExchange(ref s_EventSources, new List<WeakReference<EventSource>>(2), null);
+#if FEATURE_PERFTRACING
```

Comment:
Do we build corelib for platforms without FEATURE_PERFTRACING enabled?

Reply:
It is still conditional in some Mono TargetsBrowser builds